### ISSUE TITLE
There is no dist_test.sh file in Instance Segmentation test?

### ISSUE THREAD

**Issue by maocaixia** [microsoft/Cream#174]

**Comment by xinyuliu-jeffrey** [comment]
Hi @maocaixia ,

The script files are updated. Thanks!

**Comment by maocaixia** [comment]
Hi @xinyuliu-jeffrey ,
Thanks a lot! I test the infer time, and find the infer time of efficientvit_m0 is 19ms, but the infer time of vit_base is 7ms, It seems that there is no advantage. 
the test is in RTX4090 and Batch=1. 
vit_base: model = timm.create_model("vit_base_patch16_384", pretrained=True).
Is it normal ?

**Comment by xinyuliu-jeffrey** [comment]
@maocaixia This phenomenon is quite strange. Although we reported batch size 2048 speed, efficientvitm0 with batch size 1 still shows a satisfactory speed at my side. Have you tried the [speed_test](https://github.com/microsoft/Cream/tree/main/EfficientViT/classification#speed-test) here? Or could you please provide more details about your testing, e.g., input resolution, testing scripts, etc. Thanks!

**Comment by maocaixia** [comment]
batch size=1, what is the infer time at your side ?  what kind of gpu.
my test code:
`os.environ['CUDA_VISIBLE_DEVICES'] = '6'
from model.build import EfficientViT_M0
model = EfficientViT_M0(pretrained='efficientvit_m0')
# out = model(image)
model = model.cuda()
model = model.eval()
image = torch.randn(1,3,224,224)
image = image.cuda()
cnt = 0 
times = []
while cnt < 100:
    cnt += 1 
    t1 = time.time()
    out = model(image)
    if cnt > 30:
        times.append(time.time()-t1)
print('infer time: ', np.array(times).mean())`

**Comment by maocaixia** [comment]
@xinyuliu-jeffrey  batch size=1, what is the infer time at your side ? what kind of gpu.
my test code:
`os.environ['CUDA_VISIBLE_DEVICES'] = '6'
from model.build import EfficientViT_M0
model = EfficientViT_M0(pretrained='efficientvit_m0')
out = model(image)
model = model.cuda()
model = model.eval()
image = torch.randn(1,3,224,224)
image = image.cuda()
cnt = 0
times = []
while cnt < 100:
cnt += 1
t1 = time.time()
out = model(image)
if cnt > 30:
times.append(time.time()-t1)
print('infer time: ', np.array(times).mean())`

**Comment by maocaixia** [comment]
batch size=1, what is the infer time at your side ? what kind of gpu.
my test code:
`
os.environ['CUDA_VISIBLE_DEVICES'] = '6'
from model.build import EfficientViT_M0 
model = EfficientViT_M0(pretrained='efficientvit_m0') 
`

**Comment by maocaixia** [comment]
`os.environ['CUDA_VISIBLE_DEVICES'] = '6' 

from model.build import EfficientViT_M0 

model = EfficientViT_M0(pretrained='efficientvit_m0') out = model(image) `

**Comment by maocaixia** [comment]
`image = torch.randn(1,3,384,384)\n
image = image.cuda()\n
cnt = 0 \n
times = []\n`

**Comment by maocaixia** [comment]
···
image = torch.randn(1,3,384,384)
image = image.cuda()
cnt = 0 
times = []
···

**Comment by maocaixia** [comment]
import os 
import sys 
from timm.models import create_model
import timm
import torch 
import time
import numpy as np 

os.environ['CUDA_VISIBLE_DEVICES'] = '6'
from model.build import EfficientViT_M0
model = EfficientViT_M0(pretrained='efficientvit_m0')
# out = model(image)
model = model.cuda()
model = model.eval()

image = torch.randn(1,3,224,224)
image = image.cuda()
cnt = 0 
times = []
while cnt < 100:
    cnt += 1 
    t1 = time.time()
    out = model(image)
    if cnt > 30:
        times.append(time.time()-t1)
print('infer time: ', np.array(times).mean())

**Comment by maocaixia** [comment]
import os
import sys
from timm.models import create_model
import timm
import torch
import time
import numpy as np

os.environ['CUDA_VISIBLE_DEVICES'] = '6'
from model.build import EfficientViT_M0
model = EfficientViT_M0(pretrained='efficientvit_m0')
out = model(image)
model = model.cuda()
model = model.eval()
image = torch.randn(1,3,224,224)
image = image.cuda()
cnt = 0
times = []
while cnt < 100:
cnt += 1
t1 = time.time()
out = model(image)
if cnt > 30:
times.append(time.time()-t1)
print('infer time: ', np.array(times).mean())

**Comment by maocaixia** [comment]
@xinyuliu-jeffrey Here are my code. And what is the infer time at your side ? what kind of gpu.

**Comment by xinyuliu-jeffrey** [comment]
Hi @maocaixia , thanks for the information. I will try to test your script on my side and respond to you asap.

**Comment by maocaixia** [comment]
Hi @xinyuliu-jeffrey , thanks for your reply. 

How do you test the throughput with batch size 1? Is that use the single-model inference time calculated when batch=1, and then calculate the inverse? like infer time(batch=1, t) and throughput=1/t.

I test infer time is 19ms with batch 1 on RTX4090, so the throughput is 1000/19=52 images/s. Is it true ? 

I use the model on a hardware with small computing power only 1TOPS, so I focus more on the infer time.

**Comment by xinyuliu-jeffrey** [comment]
Hi @maocaixia , for throughput computing, please kindly refer to https://github.com/microsoft/Cream/blob/main/EfficientViT/classification/speed_test.py.

**Comment by maocaixia** [comment]
OK, thanks a lot.
